tools/cockpit.spec

```diff
@@ -93,6 +93,7 @@ Recommends: %{name}-docker = %{version}-%{release}
 Suggests: %{name}-pcp = %{version}-%{release}
 Suggests: %{name}-kubernetes = %{version}-%{release}
 Suggests: %{name}-selinux = %{version}-%{release}
+Suggests: %{name}-pkgupdates = %{version}-%{release}
```

Comment:
Okay that makes sense. But still not an trivial decision.

Our packages typically have not been named after functionality but after technology. So that multiple things with competing or different technologies can cover the same functionality ... and thus share a package alias (the ```"name"``` field in manifest.json).

> If we are going to add other mechanisms to actually update packages (like calling yum directly instead of PackageKit), these would go into this module, not a different one.

So does cockpit-ostree fould into this package or naming structure somehow? Or how does that fit into this picture? It's the same functionality with a different implementation.